crates/nu-cli/src/commands/keybindings_listen.rs

```diff
@@ -69,6 +70,32 @@ pub fn print_events(engine_state: &EngineState) -> Result<Value, ShellError> {
 
     stdout().flush()?;
     terminal::enable_raw_mode()?;
+
+    if config.use_kitty_protocol {
+        if let Ok(false) = crossterm::terminal::supports_keyboard_enhancement() {
+            println!("WARN: The terminal doesn't support use_kitty_protocol config.");
```

Comment:
In raw mode you need to supply the whole `\r\n`

```suggestion
            println!("WARN: The terminal doesn't support use_kitty_protocol config.\r");
```

Reply:
Thanks for the completion!